docs/topics/api/auth_internal.rst

```diff
@@ -20,57 +20,23 @@ Fetching the token
 ==================
 
 A fresh token, valid for 30 days, is automatically generated and added to the
-responses of the following endpoints:
+responses of the following endpoint:
 
-    * ``/api/v3/accounts/login/``
-    * ``/api/v3/accounts/register/``
+    * ``/api/v3/internal/accounts/login/start/``
     * ``/api/v3/accounts/authenticate/``
 
 A token may also be obtained through the JSON API as outlined in the
 :ref:`internal login JSON API <internal-login-json-api>` section. This is only
-accessible through the VPN and requires using the following endpoints:
+accessible through the VPN and requires using the following endpoint:
 
     * ``/api/v3/internal/accounts/login/start/``
```

Comment:
Same as above, I know you didn't add it but this line is wrong, and since `/api/v3/internal/accounts/login/` is being removed, the whole paragraph is going to need to be reworded... TBH I'm wondering if we should remove the `internal` endpoints completely, they don't make much sense anymore without the admin tools in addons-frontend.

Reply:
I'll just remove that paragraph then.

I thought we should remove the internal endpoints when the tools were removed, but the decision was they should stay (for reasons?).  Can you file a follow-up issue for the discussion?

Reply:
Back when we removed the admin tools I was in favor of keeping the APIs, but since they require either re-using a token or implementing a client with proper login flow (no other authentication mechanisms are enabled for these endpoints), meh. I'll file an issue.

Reply:
Filed https://github.com/mozilla/addons-server/issues/6568

Reply:
https://github.com/mozilla/addons-server/pull/6550/commits/c69752e36dc6eeebddfeea4b78a1ebcb91919055 for doc changes